res/xml/urdu phonetic

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Urdu phonetic" script="urdu">
+  <row>
+    <key key0="ق" key2="1" key4="esc"/>
```

Comment:
The layout contains `esc` but not `tab`.

Reply:
Thanks, I forgot about it.